Applications/Spire/Source/Ui/TableBody.cpp

```diff
@@ -142,7 +142,9 @@ struct TableBody::RowCover : Cover {
   void mount(int index) {
     auto& body = *static_cast<TableBody*>(parentWidget());
     for(auto i = 0; i != layout()->count(); ++i) {
-      get_item(i)->mount(*body.m_item_builder.mount(body.m_table, index, i));
+      auto item = body.m_item_builder.mount(body.m_table, index, i);
+      item->setFocusPolicy(Qt::NoFocus);
```

Comment:
Can you explain why the focus policy on the mounted widget is turned off?

Reply:
Cell navigation inside TableBody does not rely on the cell's focus policy or Qt's tab order list. Instead, it is handled by the focusNextPrevChild function, which takes charge of navigation within TableBody. If a cell's focus policy is set to TabFocus or StrongFocus, Qt's default behavior may interfere with TableBody’s navigation, leading to an inconsistent tab order.

Reply:
I am not sure I understand, perhaps you can help explain it to me. The item builder returns a ```QWidget``` called ```item```. ```item``` then has its focus policy set to ```NoFocus```. ```item``` is then mounted into a ```TableItem```. ```TableItem``` has a focus policy of ```StrongFocus```. When ```TableItem``` mounts ```item```, it also sets its focus proxy to ```item```, which then sets ```item```'s focus policy to ```StrongFocus```, which undoes the setting of ```item```'s focus policy to ```NoFocus```. This then makes setting the focus policy here redundant.

Did I make a mistake or misunderstand something?

Reply:
Good question. When `TableItem` mounts `item`, it doesn’t undo the the setting of `item`'s focus policy. This is because `TableItem::mount` only sets `item` as its focus proxy using`setFocusProxy(proxy)`, which does not automatically change the focus policy of `item`. 

I think a better solution is to set `TableItem` itself to `NoFocus` rather than `item`. I updated the code to set `TableItem` to `NoFocus` and adjust the focus policy when `TableItem::mount` a body, ensuring the consistency between TableItem and its body.